integration/token/tcc/dvp/views/auditor.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package views
+
+import (
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/services/assert"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/view"
+
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/ttxcc"
+)
```

Comment:
maybe adding goimports (and gofmt?) checks to the CI pipeline would help to automate to detect these formatting issues?!

Reply:
I have applied them but it is not considered a formatting issue that one.